tests/Feature/AuthorPageTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+use App\Authors\Author;
+use App\Authors\AuthorRepository;
+use App\Gists\Gistlog;
+use Carbon\Carbon;
+use Mockery\MockInterface;
+
+class AuthorPageTest extends TestCase
+{
+
+    /** @test */
+    public function a_user_can_visit_the_author_page()
+    {
+        $this->withoutExceptionHandling();
+        $gist1 = $this->createGist([
+            'id' => '234234',
+            'title' => 'My title',
+            'body' => 'my body',
+        ]);
+        $gist2 = $this->createGist([
+            'id' => '234234',
+            'title' => 'My Second title',
+            'body' => 'my second body',
+        ]);
```

Comment:
Could we make `createGist` a bit more like a factory, where it uses Faker to generate any value you didn't pass in?